sigstore/_verify.py

```diff
@@ -92,7 +104,9 @@ def verify(
 
     `signature` is a base64-encoded signature for `file`.
 
-    `cert_email` is the expected Subject Alternative Name (SAN) within `certificate`.
+    `expected_cert_email` is the expected Subject Alternative Name (SAN) within `certificate`.
+
+    `expected_cert_oidc_issuer` is the expected Organization Name of the issuer of `certificate`.
```

Comment:
I think this isn't accurate anymore. I'll push a change to fix this.

Reply:
[92c602d](https://github.com/sigstore/sigstore-python/pull/112/commits/92c602d5a49be6b0cd0901bc2b43723e33220987)